lib/Sources/BluetoothMessage/BluetoothState.swift

```diff
@@ -56,18 +64,30 @@ public actor BluetoothState {
         return peripheral
     }
 
-    public func rememberPeripheral(_ uuid: UUID) {
-        // TODO: persist known peripheral identifier
+    public func rememberPeripheral(identifier uuid: UUID) async {
+        guard let store else { return }
+        var peripheralIds = await getKnownPeripheralIdentifiers()
+        peripheralIds.insert(uuid)
+        do {
+            try await store.save(peripheralIds, for: .uuidsKey)
+        } catch {
+            log.error("Unable to persist peripheral id. \(error)")
+        }
```

Comment:
Two things:
1. opportunity to DRY this up by moving this to a method and re-using it below in `forgetPeripheral` as well
2. We should log the error text using `\(error.localizedDescription)`